components/brave_education/education_urls.cc

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_education/education_urls.h"
+
+#include <string_view>
+#include <utility>
+
+#include "base/containers/fixed_flat_map.h"
+#include "brave/components/constants/webui_url_constants.h"
+
+namespace brave_education {
+
+namespace {
+
+constexpr std::string_view PageTypeToBrowserURL(EducationPageType page_type) {
+  switch (page_type) {
+    case EducationPageType::kGettingStarted:
+      return kBraveGettingStartedURL;
+  }
+}
+
+constexpr std::string_view PageTypeToServerURL(EducationPageType page_type) {
+  switch (page_type) {
+    case EducationPageType::kGettingStarted:
+      return "https://2263.pr.bravesoftware.com/getting-started/";  //"https://browser-education.brave.com/";
```

Comment:
Should this be updated now and the comment removed or is bravesoftware.com intended to be here?

Reply:
Let me give it one last test as I just rebased on Chromium 133 - if that looks good, I'll drop that commit 😄